### PR TITLE
docs: content outline for generator docs

### LINKED ISSUE #789: [📑 Docs]: determine page outline for Generator Tool docs
### What Dev Docs changes are you proposing?

# Determine outline for Generator Tool docs
Before any TW investment is done toward adding the [Generator Tool](https://github.com/asyncapi/generator) docs in our `/website` repo, we need to agree on the content outline we'll use for this tool documentation. After the outline is agreed upon, the migration and re-writing efforts will make more sense. (cc @pratik2315 @Florence-Njeri) 

## SAMPLE Generator Tool docs outline (to get ideas rolling)
- **Overview** _(what use cases/scenarios this tool solves)_
- **Installation guide** _(download/install instructions)_
- **CLI Command reference** _(list of commands & what they do)_
- **Workflows (functionality)** _(Show us how to use the tool and it’s **functionality**. ex: generate a template, author a template)_

### URLs for new tool docs page and section hyperlinks
If we use the above sample outline, then we'd in turn end up with URLs as follows:

- `docs/tools/generator`
- `docs/tools/generator#overview`
- `docs/tools/generator#installation-guide`
- `docs/tools/generator#cli-command-reference`
- `docs/tools/generator#workflows`

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by pratik2315**
**Description**

- Created a initial outline for generator docs
- Added overview section for generator 
- Created a dummy template creation section for each programming language/framework
- I have not added the CLI section yet since parts of the current version are going to be deprecated soon, I can still add the present version and update it once new CLI is introduced, what do you guys think?!

**Related issue(s)**
Fixes #789

### CODE REVIEW

**Comment by pratik2315** [comment]
Hey @derberg @alequetzalli @Florence-Njeri !
I need your suggestions on the docs architecture I've created and the content inside introduction.md and how-to-create-a-template.md file. 
I have discussed with Florence and she is currently working on authoring.md content. After the docs structure is approved, we will shift the content present in the current files according to the new format right away! (that is, if the reallocation is really needed🤔) what do you think?

**Comment by pratik2315** [comment]
also tagging @starlightknown @Annysah @nelsonmic @thulieblack for reviews and suggestions!

**Comment by alequetzalli** [comment]
@pratik2315 First, apologies for not noticing this PR until now, thank you so much for your patience. In the future, you can also DM me if you notice I am lagging behind in seeing your PR. Just wanted you to not feel shy to remind me. 😸 

Second, the main problem to fix here is that you added **Generator Tool** docs into our `/Concepts` content bucket. That doesn't make sense, because remember that the `/Concepts` content bucket is to explain our **API spec terms**. 

Remember: Your docs PR is about the **Generator Tool**, and that is why this all needs to be moved to the `/Tools` content bucket. (you should have nothing about _tools_ inside _concepts_) 

Assuming we kept your current outline, you would have URLs along the line of...
- `docs/tools/generator/overview`
- `docs/tools/generator/filters`
- `docs/tools/generator/hooks`
- `docs/tools/generator/templates`
- etc

Screen Shot 2022-06-13 at 4 51 14 PM

**Comment by alequetzalli** [comment]
> I need your suggestions on the docs architecture I've created and the content inside introduction.md and how-to-create-a-template.md file. 

@pratik2315  Ok! I will take extra notice there, thank you for letting me know 😃 

> I have discussed with Florence as well, she is currently working on authoring.md content. 

Great! I look forward to seeing it @florence-njeri ⭐

File: docs/concepts/filters.md
Comment (alequetzalli) [inline docs/concepts/filters.md:1]: 👉🏽👉🏽 Don't forget to add to each new docs markdown page a `title` and `weight`. 

The `title` will influence the left-hand navigation in our docs. The `weight` will enable you to order these pages as you need them once we determine a set content outline. 😃 

```
---
title: 'Overview'
weight: 0
---
```

**Comment by alequetzalli** [comment]
So I am assuming this is a work in progress, but I was wondering (confused? 🤔 😆) about the `` and `` files. What were these **empty** files added for? 

Screen Shot 2022-06-13 at 6 20 21 PM

File: docs/tutorials/how-to-create-a-template.md
Comment (alequetzalli) [inline docs/tutorials/how-to-create-a-template.md:1]: Ok, so remember how you're adding a NEW **sub-section** to Tools (`/docs/tools/generator`)?  Thus, you will also be adding a `_section.md` to that as  **sub-section** similar to what we see here...

section markdown files in our docs

For **Generator Tool** _tutorials_, you should see a content outline and URL patterns such as...

- `docs/tutorials/generator-tool/installation-guide`
- `docs/tutorials/generator-tool/how-to-create-a-template`
- etc.

Comment (alequetzalli) [reply docs/tutorials/how-to-create-a-template.md:1]: P.S. This tutorial you're making is looking great!! I know your work is going to help our community a lot, thank you so much for your contributions 🎉✨✨✨✨

File: docs/concepts/hooks.md
Comment (alequetzalli) [inline docs/concepts/hooks.md:18]:
```suggestion
Most basic modules with hooks look like this:
```

File: docs/concepts/index.md
Comment (alequetzalli) [inline docs/concepts/index.md:3]:
```suggestion

```
So I would remove this header because it seems more playful; something I'd see in a blog post for example but not necessarily documentation. 

Also, let's have the first header of `Overview` be the sole header for this introduction bit.

**Comment by alequetzalli** [comment]
@pratik2315 @florence-njeri Ok, assuming this is the only content you need to add, here's the outline I'm proposing so far... (of course, you should veto me in the order if you know best)...

**NOTE**: let's put the **installation guide** inside the `TOOLs` content bucket, not in _tutorials_. ❤️😄✌🏽 In addition, should we end up with a CLI doc, that too should be in our `TOOLs` content bucket, not in _tutorials_.

### `Tools` content bucket URLs:
- `docs/tools/generator/overview`
- `docs/tools/generator/installation-guide`
- `docs/tools/generator/cli-command-reference` **(ONLY if this isn't deprecated)**
- `docs/tools/generator/filters`
- `docs/tools/generator/hooks`
- `docs/tools/generator/parser`
- `docs/tools/generator/templates`
- etc

For this section, I also want you to think about **order** of your new pages. Does it matter to the user the order of the following pages? 
- filters
- hooks
- parsers
- templates

If **order** does matter, then make sure you add page `weights` that reflect the order in which these pages should be read. 👍🏽

Screen Shot 2022-06-13 at 6 13 13 PM



### `Tutorials` content bucket URLs:
- `docs/tutorials/generator/overview`
- `docs/tutorials/generator/how-to-create-a-template`
- etc


@pratik2315 Let me know if this doesn't make sense! We can always hop on a call. 📞 😄

**Comment by alequetzalli** [comment]
@florence-njeri @pratik2315 Would like to recommend you subscribe to the FREE plan of Grammarly: https://www.grammarly.com/plans

The truth is that I will be doing this for all of our work, so it's good for budding TWs to get in the habit of using these tools too. 😄✌🏽

You need to run all your TW drafts through a program like this to catch your first round of grammatical errors, typos, etc. :slightly_smiling_face: I promise this will make your life easier! 

Screen Shot 2022-06-13 at 4 29 04 PM

**Review by alequetzalli — CHANGES_REQUESTED** [review]
Ok @pratik2315 @florence-njeri, I have added my preliminary review for the **outline**, URL structures, and some important info for adding page weights/sub-sections. 

I have not conducted an editorial review of your written content yet because I agree with Pratik that the first step is to agree on the new outline. 

I look forward to your next draft and response to my feedback!  This is great work so far, I am so excited to get to work and learn with you! 🤩🤩🤩🤩

**Comment by pratik2315** [comment]
And idk how I missed your

> Ok @pratik2315 @Florence-Njeri, I have added my preliminary review for the **outline**, URL structures, and some important info for adding page weights/sub-sections.
> 
> I have not conducted an editorial review of your written content yet because I agree with Pratik that the first step is to agree on the new outline.
> 
> I look forward to your next draft and response to my feedback! This is great work so far, I am so excited to get to work and learn with you! 🤩🤩🤩🤩

I just saw your responses when I opened this PR, I have no idea how I missed your feedback😓 For some reason, Github didnt send me any notifications about your comments! Yes the content inside needs tons of modifications, I have yet to discuss and confirm with @derberg about the technical terms used in the docs😁 I'll be running my docs through Grammarly first! Thanks a bunchh for your feedback🤩🤩 I'll respond to each of your comments in a moment. And as always, I look forward to work and learn with you too😆😆

**Comment by pratik2315** [comment]
> @pratik2315 First, apologies for not noticing this PR until now, thank you so much for your patience. In the future, you can also DM me if you notice I am lagging behind in seeing your PR. Just wanted you to not feel shy to remind me. 😸
> 
> Second, the main problem to fix here is that you added **Generator Tool** docs into our `/Concepts` content bucket. That doesn't make sense, because remember that the `/Concepts` content bucket is to explain our **API spec terms**.
> 
> Remember: Your docs PR is about the **Generator Tool**, and that is why this all needs to be moved to the `/Tools` content bucket. (you should have nothing about _tools_ inside _concepts_)
> 
> Let's pretend we kept your current outline; you'd end up with URLs such as...
> 
> * `docs/tools/generator/overview`
> * `docs/tools/generator/filters`
> * `docs/tools/generator/hooks`
> * `docs/tools/generator/templates`
> * etc
> 
> Screen Shot 2022-06-13 at 4 51 14 PM

Ohh okay! My mistake, I thought we were having second concepts file under tools🤦‍♂️ thanks for clarifying! I'll move all of those files under tools asap!

File: docs/concepts/filters.md
Comment (pratik2315) [reply docs/concepts/filters.md:1]: okay got it!👍

File: docs/concepts/hooks.md
Comment (pratik2315) [reply docs/concepts/hooks.md:18]: oops!!

**Comment by pratik2315** [comment]
> @pratik2315 @Florence-Njeri Ok, assuming this is the only content you need to add, here's the outline I'm proposing so far... (of course, you should veto me in the order if you know best)...
> 
> **NOTE**: let's put the **installation guide** inside the `TOOLs` content bucket, not in _tutorials_. ❤️😄✌🏽 In addition, should we end up with a CLI doc, that too should be in our `TOOLs` content bucket, not in _tutorials_.
> 
> ### `Tools` content bucket URLs:
> * `docs/tools/generator/overview`
> * `docs/tools/generator/installation-guide`
> * `docs/tools/generator/cli-command-reference` **(ONLY if this isn't deprecated)**
> * `docs/tools/generator/filters`
> * `docs/tools/generator/hooks`
> * `docs/tools/generator/parser`
> * `docs/tools/generator/templates`
> * etc
> 
> For this section, I also want you to think about **order** of your new pages. Does it matter to the user the order of the following pages?
> 
> * filters
> * hooks
> * parsers
> * templates
> 
> If **order** does matter, then make sure you add page `weights` that reflect the order in which these pages should be read. 👍🏽

I may be wrong but order does matter (for me at least). I think its best to explain the concepts in the order in which they are used in Generator library. When I look at it from user point of view, it seems logical😄 but I may be incorrect, What do you think @derberg @Florence-Njeri ?!
> 
> Screen Shot 2022-06-13 at 6 13 13 PM
> 
> ### `Tutorials` content bucket URLs:
> * `docs/tutorials/generator/overview`
> * `docs/tutorials/generator/how-to-create-a-template`
> * etc
> 
> @pratik2315 Let me know if this doesn't make sense! We can always hop on a call. 📞 😄

**Comment by pratik2315** [comment]
> Hi Pratik, [asyncapi-context](https://github.com/asyncapi/generator/pull/793#issuecomment-1157598832), I think if I remember correctly, it was just context (not asyncapi-template)
> 
> [Why would we have an overview again in tutorials](https://github.com/asyncapi/generator/pull/793#issuecomment-1157612775) I think about it as an introduction to the generator tool just before the user checks how to create a template. Alternatively, instead of adding an overview section in the tutorials, I can link the tools/generator/overview in the how to create template tutorial.
> 
> I'll add a note section after the introduction. e.g
> 
> > **_NOTE:_**  This tutorial assumes you have some background knowledge of what the generator tool is and how to use the available templates. If you are unfamiliar with the generator tool, you can checkout our [tools](link to the docs)documentation

thats what I was thinking😁 seems like a perfect way to link users with needed information as you said

**Review by derberg — COMMENTED** [review]
My suggestions for this PR:
- let's focus only on structure and the content we already have
- new content in separate PRs for easier review and easier engagement of other maintainers
- if new structure assumes new file with no content, just put there a note that content will be added later
- this PR should remove `authoring.md` and before we end up work and get this content in the website, let's just have a simple README in `docs` with ordered list of links to different new markdown files, like navigation that long term we will see in the website
- I recommend putting all in `docs` in flat and for now forget about `_section.md` as this is not much important for now. We might handle "structure" and website-specific files creation in GitHub Action workflow, who knows 🤷🏼 


Critical is to start restructuring, splitting into files, identifying what documentation is missing, and adding it. Later the rest.

Thoughts?

File: docs/tutorials/how-to-create-a-template.md
Comment (alequetzalli) [reply docs/tutorials/how-to-create-a-template.md:1]: > > For Generator Tool tutorials, you should see a content outline and URL patterns such as...
> 
> 
> 
> do you mean that tutorials for generator will not be in "tools" bucket, next to other generator-related docs but in "tutorials"?

@derberg You got it

**Review by Florence-Njeri — APPROVED** [review]

**Comment by pratik2315** [comment]
Hey there @derberg and @alequetzalli !
Me and @Florence-Njeri  have worked on some changes👨‍💻👩‍💻. I have also restructured the content from authoring.md and got it reviewed by Florence since a lot of her previous work was focused on authoring.md. The [readme](https://github.com/pratik2315/generator/tree/gsod1-pratik/docs#readme)in the docs contains the order in which we have assigned the weights to each section. Can you pls review the these changes😁

File: docs/asyncapi-context.md
Comment (derberg) [inline docs/asyncapi-context.md:6]:
```suggestion
> This document is under construction.
```

please do it as quite, and maybe like this :)

**Review by derberg — CHANGES_REQUESTED** [review]
great job so far folks!!!

I left comments, also please:
- remove https://github.com/asyncapi/generator/blob/master/docs/templates-recipes.md as it is also not really maintained
- update https://github.com/asyncapi/generator#how-to-create-a-template
- also we will need to fix links in other repos https://github.com/search?q=org%3Aasyncapi+authoring.md&type=code as because after merging this PR, CI will fail because of broken links

File: docs/README.md
Comment (alequetzalli) [inline docs/README.md:1]: @Florence-Njeri @pratik2315   Why is everything under the main `/docs` directory? Remember, we are documenting the **tool**, Generator Tool; that means you should be adding your content under `docs/tools/generator`.  

If we kept your latest outline topic example, you'd end up with URLs such as...
- `docs/tools/generator/overview`
- `docs/tools/generator/installation-guide`
- `docs/tools/generator/asyncapi-spec-file`
- `docs/tools/generator/configuration-file`
- `docs/tools/generator/configuration-file#special-parameters`
- `docs/tools/generator/templates`
- `docs/tools/generator/file-templates`
- `docs/tools/generator/authoring-templates#special-file-names`
- etc...

File: docs/README.md
Comment (alequetzalli) [inline docs/README.md:3]: I wanted to make sure we remember the [comment discussion about the outline starting with `overview` and `installation-guide`](https://github.com/asyncapi/generator/pull/793#issuecomment-1154620376). 

I'm not sure about **introduction** and going straight into **asyncapi spec file**.  What do you think about the following instead?
- `docs/tools/generator/overview`
- `docs/tools/generator/installation-guide`

File: docs/README.md
Comment (alequetzalli) [inline docs/README.md:11]: Hmm **Common Assumptions** doesn't sound right for a doc's site outline title. 🤔 

This sounds like something we either need to (a) find a diff title; or (b) incorporate this section into another one.

What do you think @pratik2315 @Florence-Njeri ?

**Review by alequetzalli — CHANGES_REQUESTED** [review]
Left feedback for the outline and content bucket sections. 

The new outline is looking better already, we just need to fix the lack of content buckets.
🎉

File: docs/README.md
Comment (pratik2315) [reply docs/README.md:3]: Overview seems good. Maybe we can have overview and installation-guide in the same file🤔 Since installation guide is just few lines?

**Comment by alequetzalli** [comment]
> @alequetzalli
> 
> > The new outline is looking better already, we just need to fix the lack of content buckets.
> 
> Please have a look at [#793 (review)](https://github.com/asyncapi/generator/pull/793#pullrequestreview-1015419932). We want to first focus on content, identify missing parts, update and add new ones. Then once we have the content, we can actually focus on a structure. Basically one step at a time, one PR at a time.

thank goodness, I was worried no one had understood the content buckets 😂😂

That sounds great, honestly, with all the comments here that one got lost so now, I read it and feel up to speed. ✌🏽

@derberg

**Comment by Florence-Njeri** [comment]
> great job so far folks!!!
> 
> I left comments, also please:
> 
> * remove https://github.com/asyncapi/generator/blob/master/docs/templates-recipes.md as it is also not really maintained
> * update https://github.com/asyncapi/generator#how-to-create-a-template
> * also we will need to fix links in other repos https://github.com/search?q=org%3Aasyncapi+authoring.md&type=code as because after merging this PR, CI will fail because of broken links

@derberg @alequetzalli I made changes to fix links in the other repos

1. https://github.com/asyncapi/website/pull/845
2. https://github.com/asyncapi/ts-nats-template/pull/478
3. https://github.com/asyncapi/dotnet-nats-template/pull/277
4. https://github.com/asyncapi/template-for-generator-templates/pull/190

File: docs/README.md
Comment (alequetzalli) [reply docs/README.md:3]: Hmm I see what you mean! 🤔 So, I thought about it and it seems off, because then in the **left-hand** navigation of docs the user would only see "overview" but not the "installation guide" because headers don't show up in that area.

How do you feel about doing it this way, instead? (It would also mirror consistency with what we're doing in our **Tutorials** now, as you can see in our [updated IA outline for Streetlights tutorial](https://www.asyncapi.com/docs/tutorials/streetlights).)
- `tools/generator/set-up-generator-tool`
- `tools/generator/set-up-generator-tool#introduction`
- `tools/generator/set-up-generator-tool#background-context`
- `tools/generator/set-up-generator-tool#installation-guide`

Comment (alequetzalli) [reply docs/README.md:3]: I also wanted you to take a look at a similar scenario and how **Gatsby** handled it, because I think it will visually also help us for our work in this project. 😀
Screen Shot 2022-06-29 at 1 27 53 PM

File: docs/authoring-templates.md
Comment (alequetzalli) [inline docs/authoring-templates.md:12]:
```suggestion

```

Like I said before, this header isn't right for a docs site. 😀

Let's make sure we think about this one in a different way, such as the following solution I can help propose. I think you need to turn this into one of our `Remember` React components, such as the one we have in our [Getting Started: Servers](https://www.asyncapi.com/docs/tutorials/getting-started/servers) tutorial. Then, all you have to do is determine in which current section it makes most sense to be in. It could also be that you need to turn this list into more than one `Remember` React component, adding individual items to where they most apply.

File: docs/README.md
Comment (alequetzalli) [reply docs/README.md:3]: Other than this particular point we're pondering, the rest of the outline you have proposed here seems to work pretty great! 

Of course, make sure that Lukasz signs off on the order of the pages, since he will know best technically the correct order needed for users.

File: docs/README.md
Comment (alequetzalli) [reply docs/README.md:1]: @derberg can you educate me about why `File Templates` goes so far at the end, instead of right after the `Templates` and `Authoring Template` pages? In my mind, I am missing the connection and I think I must need more context. 😀

Comment (pratik2315) [reply docs/README.md:1]: My thoughts exactly😅

File: docs/authoring-templates.md
Comment (pratik2315) [reply docs/authoring-templates.md:12]: Totally forgot about this! I guess renaming the Common assumptions to Remember would fit the architecture as well🤔 And both "Remember" and "Common assumptions" kind of indicate the same thing, what'd you think @derberg @Florence-Njeri 😁

File: docs/README.md
Comment (pratik2315) [reply docs/README.md:3]: The main reason I was refraining from using the word "Overview" is because we already have an Overview section in the Tools, here it is:

Screenshot (86)

If we include the generator overview again, we will have two overview's in the same section and most likely one below the other🤔
your suggested outlines and the gatsby reference sound awesome 
- tools/generator/set-up-generator-tool
- tools/generator/set-up-generator-tool#introduction
- tools/generator/set-up-generator-tool#background-context
- tools/generator/set-up-generator-tool#installation-guide
- tools/generator/set-up-generator-tool#summary

But I have to take @derberg 's opinion on whether these outlines sound technically correct or not😅

File: docs/README.md
Comment (pratik2315) [reply docs/README.md:1]: Even in the original authoring.md, the section "File templates" came right after authoring-templates🤔

File: docs/README.md
Comment (derberg) [reply docs/README.md:3]: remember folks, try not to think to much about the website, content first 😄 

I'm ok with `introduction` or `overview`. There is no standard out there, people do it differently, and I, as a user, stopped paying attention to the `title`. Just leave `Introduction` and we can always improve at the end, current README navigation is there only temporarily until we integrate with the website

File: docs/authoring-templates.md
Comment (derberg) [reply docs/authoring-templates.md:12]: `Common assumptions` was an unfortunate name from the very beginning, but `Remember` and note is also not very good in this case. This section explains in short some basic information about templates, like intro to features. We could later think on transforming it into a kind of FAQ like: 
- Q: `how can I write a code that performs some modifications in the template after generation or before generation process` 
- A: Please use hooks -> link 


anyway not in scope of this PR, just make a note 😉 

so for now, best, just remove the header, and `Special file names` turn into a separate file, and put it at the end of the list in the navigation

File: docs/README.md
Comment (derberg) [reply docs/README.md:1]: yeah, it might be so called `opinionated` decision 😄 

`File Templates` for me are needed for the user that already understood the basics of writing a generator template and the user is ready to write a code generator, as `File Templates` functionality was created mainly for code generation use case, where you need to generate multiple files out of a single file

File: docs/README.md
Comment (pratik2315) [reply docs/README.md:3]: Got it! Thanks @derberg 
Thanks @alequetzalli for the Gatsby reference! I checked out their documentation and that section about "setting up" would help the generator documentation greatly in future PR's

File: docs/authoring-templates.md
Comment (Florence-Njeri) [reply docs/authoring-templates.md]: @derberg can the common assumptions be considered as features of a template?

Comment (derberg) [reply docs/authoring-templates.md]: kind of 😄 because it is a mix of features and common information....really bad section 😄 

sometimes it talks about features, sometimes just states facts, like that template files should be in `template` dir 😄

File: docs/README.md
Comment (alequetzalli) [reply docs/README.md:3]: > Thanks @alequetzalli for the Gatsby reference! I checked out their documentation and that section about "setting up" would help the generator documentation greatly in future PR's

@pratik2315 @Florence-Njeri Great! So it sounds like we agree with this new outline moving forward!! 

I **completely agree with you** that using the term _overview_ in this section would be over use. That's why I am glad that we both agree that the word _introduction_ is better suited for this section.

File: docs/authoring-templates.md
Comment (alequetzalli) [reply docs/authoring-templates.md]: @derberg @Florence-Njeri @pratik2315 

Nope, sorry kidos.. no Q/A here. Adding an FAQ section is a very simplistic and unfortunately lazy way to handle TW problems. 

We really have to think about an appropriate way to add this information and if it is not with a remember react component, then we still need to plan where in the outline this information fits 
💜👍🏽👍🏽

Comment (pratik2315) [reply docs/authoring-templates.md]: No FAQ's!!

**Comment by pratik2315** [comment]
Hey @derberg @alequetzalli ! I think we have made all the needed changes🤔 Can you check if I missed something😅 
- As per Lukasz's suggestion, I have removed the common assumptions header for now. The content inside Common assumptions is in the same file and not removed. Since, the header discussion is out of scope for this PR and will be discussed in the next one, I thought it best not to remove any of the content stuff.
- Special file names is a different section and at the end of the navigation as per this [comment](https://github.com/asyncapi/generator/pull/793#discussion_r910859833)😀

Can you pls review if any other changes are needed and point out any mistakes, Thanks a bunch😀

File: docs/asyncapi-file.md
Comment (derberg) [inline docs/asyncapi-file.md:6]:
```suggestion
> This document is under construction.
```

just do those as a markdown quote

File: README.md
Comment (derberg) [inline README.md:289]:
```suggestion
- Read [the documentation](docs/README.md)
```

just link to the main docs as all of them are about writing a template

**Comment by derberg** [comment]
@magicmatatjahu @jonaslagoni @fmvilas yo folks, any objections to getting this PR merged?

it is the first step, just restructuring and splitting what we have, no changes in content itself (like new or rewrite). Followup PRs will be about adding missing documentation, missing reference docs and missing tutorials. And then having it on the website.

it is all part of Google Season of Docs and assignment for @Florence-Njeri and @pratik2315

**Comment by pratik2315** [comment]
hmm seems like 1 check is failing and im not sure what is that. any idea @derberg @Florence-Njeri ?🤔

**Comment by pratik2315** [comment]
I just checked and saw that this 1 test is failing for all the pending pr's in the generator repository🤔👩‍💻

**Comment by derberg** [comment]
@pratik2315 do not worry about it, it is related to https://github.com/asyncapi/generator/issues/8. You did nothing wrong

**Review by jonaslagoni — APPROVED** [review]
👍

**Comment by pratik2315** [comment]
> @pratik2315 do not worry about it, it is related to #8. You did nothing wrong

Ah i see! Thanks😃

**Review by fmvilas — APPROVED** [review]
:shipit:

**Review by magicmatatjahu — APPROVED** [review]

**Review by derberg — APPROVED** [review]

**Comment by derberg** [comment]
@pratik2315 @Florence-Njeri congrats folks 🥳

**Pull request merged**
